cpp/src/arrow/compute/exec/test_util.cc

```diff
@@ -372,22 +374,12 @@ static inline void PrintToImpl(const std::string& factory_name,
     for (const auto& agg : o->aggregates) {
       *os << agg.function << "<";
       if (agg.options) PrintTo(*agg.options, os);
+      *os << agg.target.ToString() << "<";
```

Comment:
I'm not sure this is correct.  Ideally we would have something like...

```
aggregates=[{function=hash_sum<>,target=a,name=sum(a)}]
```

I think the way it is now it would print:
```
aggregates={hash_sum<a<sum_a<>,},
```